interop/kotlinx-metadata/api/kotlinx-metadata.api

```diff
@@ -1,19 +1,19 @@
 public final class com/squareup/kotlinpoet/metadata/KotlinPoetMetadata {
-	public static final fun readKotlinClassMetadata (Lkotlin/Metadata;)Lkotlinx/metadata/jvm/KotlinClassMetadata;
-	public static final fun toKmClass (Ljava/lang/Class;)Lkotlinx/metadata/KmClass;
-	public static final fun toKmClass (Ljavax/lang/model/element/TypeElement;)Lkotlinx/metadata/KmClass;
-	public static final fun toKmClass (Lkotlin/Metadata;)Lkotlinx/metadata/KmClass;
-	public static final fun toKmClass (Lkotlin/reflect/KClass;)Lkotlinx/metadata/KmClass;
+	public static final fun readKotlinClassMetadata (Lkotlin/Metadata;Z)Lkotlinx/metadata/jvm/KotlinClassMetadata;
+	public static final fun toKmClass (Ljava/lang/Class;Z)Lkotlinx/metadata/KmClass;
+	public static final fun toKmClass (Ljavax/lang/model/element/TypeElement;Z)Lkotlinx/metadata/KmClass;
+	public static final fun toKmClass (Lkotlin/Metadata;Z)Lkotlinx/metadata/KmClass;
+	public static final fun toKmClass (Lkotlin/reflect/KClass;Z)Lkotlinx/metadata/KmClass;
 }
```

Comment:
None of these functions have KotlinPoet types as parameter types or return types. Do they need to be public?

Reply:
We've historically offered a bit of a superset of APIs here with some helpers to make KM usage easier. It's become less necessary now though that the API on their side has caught up. Let me take a pass at cleaning these up in a followup PR?

Reply:
Sounds good